packages/react-magma-dom/src/components/CharacterCounter/CharacterCounter.tsx

```diff
@@ -6,6 +6,10 @@ import { I18nContext } from '../../i18n';
 
 export interface CharacterCounterProps
   extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Allows screenreaders to iterate on the default state of the title.
+   */
+  id?: string;
```

Comment:
I think the `id` should be required. As far as the description, I'm not sure what "iterate on the default state of the title" means 😕 . I think we use it as an identifier to associate the character counter with the input field or message, right?